### PR TITLE
Feat Add visitor

### PR DESCRIPTION
**Pull request by alex596**
* Feat : Add possibility to set path on configuration
* Feat : Add possibility to set customUserAgent on configuration
* Feat : Add possibility to set crashDetection on configuration
* Feat : Add possibility to set sessionBackgroundDuration on configuration
* Feat : Add possibility to set ignoreLimitedAdvertisingTracking on configuration
* Feat : Add possibility to set sessionBackgroundDuration on configuration
* Feat : Add possibility to set privacyDefaultMode on configuration
* Feat : Add possibility to set offlineEncryptionMode on configuration
* Feat : Add possibility to set offlineStorageMode on configuration
* Feat : Add possibility to set storageLifetimePrivacy on configuration
* Feat : Add possibility to set storageLifetimeUser on configuration
* Feat : Add possibility to set storageLifetimeVisitor on configuration
* Feat : Add possibility to set visitorStorageMode on configuration
* Feat : Add possibility to set visitorIdTypeMode on configuration
* Feat : Add possibility to set visitorId on configuration

[Documentation for configuration](https://developers.atinternet-solutions.com/piano-analytics/data-collection/sdks/ios-swift#configuration)

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-group 'com.plugin.example.piano_analytics_plugin'
+group 'com.plugin.piano_analytics_plugin'
 version '1.0-SNAPSHOT'
```
Comment (Coyote79) [inline android/build.gradle:2]: Set the version

File: android/build.gradle [hunk 2]
```diff
@@ -49,4 +49,5 @@ android {
 
 dependencies {
     implementation 'io.piano:analytics:3.2.1'
+    //implementation 'io.piano.android:analytics:3.4.0'
```
Comment (Coyote79) [inline android/build.gradle:52]: Maybe add a comment to indicate a major impact ?

Comment (alex596) [reply android/build.gradle:52]: Hi, I wanted to keep track of the future evolution of the plugin (1.0.0). Besides, I'm going to create a ticket with this in mind.

Comment (alex596) [reply android/build.gradle:52]: Here is the ticket for next evol: https://github.com/alex596/piano_analytics_plugin/issues/10

File: android/build.gradle (hunk 1 above)
Comment (alex596) [reply android/build.gradle:2]: The version of this plugin is set to pubspec.yaml. I just refacto for delete the exemple folder

**Pull request merged**